wordpress-mobile/WordPress-iOS
# Integrates Aztec v1.0.0-beta.11.1

Integrates the latest fixes from Aztec.

Includes some important bugfixes for iOS 11.

Most changes affect formatting.

**How to test:**

- Test styling
- Test header styles
- Test editing lists with multiple bullets.
- Test simple styling in iOS 11... make sure style persist after the first character is typed.

PS: since this was a bit rushed out, I take responsibility for any issues that arise from this integration.  I spent a good amount of time today testing it myself.  The fixes for iOS 11 felt important enough to make this exception.
